src/vmm/src/devices/virtio/queue.rs

```diff
@@ -625,14 +625,7 @@ impl Queue {
     }
 
     /// Puts an available descriptor head into the used ring for use by the guest.
-    pub fn add_used<M: GuestMemory>(
-        &mut self,
-        mem: &M,
-        desc_index: u16,
-        len: u32,
-    ) -> Result<(), QueueError> {
-        debug_assert!(self.is_valid(mem));
-
```

Comment:
Can we merge the commit that does this change with the one that switches from `write_used_ring` to `used_ring_ring_set`?

Reply:
why? these are 2 different changes?

Reply:
Also can you give commits names? It is very annoying trying to find commit where you left the comment.